trading_calendars/calendar_helpers.py

```diff
@@ -44,28 +58,63 @@ def is_open(opens, closes, minute_val):
             return False
 
 
-def compute_all_minutes(opens_in_ns, closes_in_ns):
+def is_betweeen_open_and_close(opens, closes, minute_val):
     """
-    Given arrays of opens and closes, both in nanoseconds,
-    return an array of each minute between the opens and closes.
+    Works like `is_open`, ignoring the existence of breaks
     """
-    deltas = closes_in_ns - opens_in_ns
-
-    # + 1 because we want 390 mins per standard day, not 389
-    daily_sizes = (deltas // NANOSECONDS_PER_MINUTE) + 1
-    num_minutes = daily_sizes.sum()
+    open_idx = np.searchsorted(opens, minute_val)
+    close_idx = np.searchsorted(closes, minute_val)
+    if open_idx != close_idx:
+        # if the indices are not same, that means we are within a session
+        return True
```

Comment:
This looks correct to me, but what do you think about merging this function back into `is_open` and giving it an `ignore_breaks` parameter, so that we're not duplicating the non-trivial logic of `is_open`?

This different block would turn into:
```python
if ignore_breaks:
    return True
else:
    # break-specific logic above
```